docs/migrate/from-hono.md

```
@@ -82,15 +82,13 @@ Both use a single `Context` parameters to handle request and response, and retur
 
 const app = new Hono()
-
-app.get('/', (c) => {
-    return c.text('Hello World')
-})
-
-app.post('/id/:id', (c) => {
-	c.status(201)
-    return c.text(req.params.id)
-})
+	.get('/', (c) => {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)